packages/edit-site/src/components/page-pages/index.js

```diff
@@ -31,6 +32,11 @@ export default function PagePages() {
 			field: 'date',
 			direction: 'desc',
 		},
+		fields: {
+			// All fields are visible by default, so it's
+			// better to keep track of the hidden ones.
+			hidden: [ 'date' ],
```

Comment:
Do you think we should move this to the top `hiddenFields` instead of `fields.hidden`?

Reply:
I expect to keep more info about fields in follow ups, but yes. We could move it for now. I kept it because I had the false assumption about the `hidable` prop before.